packages/aws_common/pubspec_overrides.yaml

```diff
@@ -0,0 +1,4 @@
+# melos_managed_dependency_overrides: amplify_lints
+dependency_overrides:
```

Comment:
These are tracked until mono_repo support expands or the pkgs are integrated into CircleCI